subsys/espi/ndsx_espi.c

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/espi.h>
+
+#include "x86_non_dsx.h"
+
+static const struct device *espi_dev;
+static struct espi_callback espi_bus_cb;
+static struct espi_callback espi_chan_cb;
+static struct espi_callback espi_vw_cb;
+
+static void espi_bus_reset(void)
+{
+	/* If SOC is up toggle the PM_PWRBTN pin */
+	if (power_gpio_sig_get(POWER_PCH_EC_SLP_SUS_N)) {
+		LOG_INF("Toggle PM PWRBTN");
+
+		gpio_set_level(EC_PCH_PWRBTN_N, 0);
+		k_msleep(POWER_EC_PCH_PM_PWRBTN_DELAY_MS);
+		power_gpio_sig_set(EC_PCH_PWRBTN_N, 1);
+	}
+}
+
+static void espi_vw_handler(struct espi_event *event)
```

Comment:
The code in this function sees like it it target to the power sequencing part. I think this code should go in the power sequence folder (in the same file as above espi power sequencing stuff)

Reply:
ACK